src/config.rs

```diff
@@ -273,6 +288,7 @@ struct ConfigParser {
     mjpeg_paths_file: String,
     log_file: String,
     discovery: bool,
+    discovery_whitelist: Vec<String>,
```

Comment:
Is there a reason why `HashSet<String>` can't be used right from the beginning?

Reply:
There is no special reason for it. Incremental construction of a hash set has slightly higher overhead in terms of binary size and used memory in runtime but it'd be a micro optimization.